discretize/MeshIO.py

```diff
@@ -44,6 +44,63 @@ def readCellLine(line):
         # Make the mesh
         tensMsh = TensorMesh([h1, h2, h3], x0)
         return tensMsh
+    
+    @classmethod
+    def readUBC_DC2DMesh(TensorMesh,fileName):
+        """
+        Read UBC GIF 2DTensor mesh and generate 2D Tensor mesh in simpeg
+
+        :param string fileName: path to the UBC GIF mesh file
+        :rtype: TensorMesh
+        :return: SimPEG TensorMesh 2D object
+
+        Created on Thu Nov 12 13:14:10 2015
+
+        @author: dominiquef
+
+        """
+
+        fopen = open(fileName, 'r')
+
+        # Read down the file and unpack dx vector
+        def unpackdx(fid, nrows):
+            for ii in range(nrows):
+
```

Comment:
would you mind opening up this file in a text editor with a pep8 linter? there is a bit of whitespace cleanup that can be done (eg within the function there should be at most one line between each code statement). To get up and running with a sublime pep8 linter, please see https://github.com/SublimeLinter/SublimeLinter-pep8